src/hooks/useLocalStorage.js

```diff
@@ -1,9 +1,9 @@
 import { useState } from 'react';
 
-export function useLocalStorage(key, initialValue) {
+export function useLocalStorage(initialValue = '') {
   const [storedValue, setStoredValue] = useState(() => {
     try {
-      const item = window.localStorage.getItem(key);
+      const item = window.localStorage.getItem('LIST_TOKEN_V1');
```

Comment:
I believe `LIST_TOKEN_V1` could be a parameter `useLocalstorage` because you use it in multiple places, and might help us the maintainability.

How can you see it?